gitops/snapshot_predicate.go

```diff
@@ -23,3 +23,22 @@ func IntegrationSnapshotChangePredicate() predicate.Predicate {
 		},
 	}
 }
+
+// SnapshotTestAnnotationChangePredicate returns a predicate which filters out all objects except
+// snapshot annotation "test.appstudio.openshift.io/status" is changed for update events.
+func SnapshotTestAnnotationChangePredicate() predicate.Predicate {
+	return predicate.Funcs{
+		CreateFunc: func(createEvent event.CreateEvent) bool {
+			return true
```

Comment:
I think we don't want to react to the Snapshot creation in the statusreport controller, but I might be missing something.

Reply:
Thanks. I copied but just update the UpdateFunc. Updated.